ipa-core/src/cli/mod.rs

```diff
@@ -1,6 +1,8 @@
 #[cfg(feature = "web-app")]
 mod clientconf;
 mod csv;
+#[cfg(all(feature = "test-fixture", feature = "web-app", feature = "cli"))]
+pub mod hpke;
```

Comment:
this module name is confusing to me - I assumed that it provides the actual implementation of HPKE spec, which is not the case. Maybe it needs a better name

Reply:
that's fair. `hpke_util`?

Reply:
`crypto` maybe? HPKE may give false suspicions to readers that it is somehow related to the existing `hpke` module, but there is no HPKE-specific functionality here, just encrypt/decrypt